src/dbnode/encoding/proto/round_trip_prop_test.go

```diff
@@ -156,7 +156,7 @@ func TestRoundTripProp(t *testing.T) {
 		}
 
 		stream := enc.Stream()
-		if stream == nil {
+		if !ok {
```

Comment:
Hm, where did this `ok` come from?  Shouldn't the line above be `stream, ok := enc.Stream()`?